waddrmgr/db.go

```diff
@@ -549,14 +581,14 @@ func putNumAccounts(tx walletdb.Tx, numAccounts uint32) error {
 // the common parts.
 func deserializeAddressRow(addressID, serializedAddress []byte) (*dbAddressRow, error) {
 	// The serialized address format is:
-	//   <addrType><account><addedTime><syncStatus><rawdata>
+	//   <addrType><account><addedTime><syncStatus><watchingOnly><rawdata>
```

Comment:
Updated to only use the `watchingOnly` flag and removed `adtWatching` type. Importing "loose" public keys and P2SH hashes works using the `importpubkey` and `importaddress` commands. Now working on importing public key chains. 

Now working on importing BIP0032 public key chains.